admin-console/src/test/java/com/agiletec/apsadmin/portal/TestPageAction.java

```diff
@@ -49,7 +49,6 @@
 import java.util.Date;
 import java.util.Set;
 import org.entando.entando.ent.exception.EntException;
-import org.junit.jupiter.api.Assertions;
```

Comment:
this file has the wrong name, `Test` must be the last word in a filename, i remember some exclusions but it should be possible to find a name that matches our rule ...

Reply:
Maybe it's better to create a ticket to fix the naming convention in whole source code